### PR TITLE
Smearing kernel

### PR DESCRIPTION
**Pull request by RenataKrupczak**
this is linked to issue #293

### CODE REVIEW

File: src/sparkx/EventCharacteristics.py [hunk 1]
```diff
@@ -321,6 +321,7 @@ def generate_eBQS_densities_Milne_from_OSCAR_IC(
         n_sigma_y: Union[float, int],
         n_sigma_z: Union[float, int],
         sigma_smear: Union[float, int],
+        kernel: str,
```
Comment (NGoetz) [inline src/sparkx/EventCharacteristics.py:324]: kernel should have a default value. "gaussian", I'd propose

File: src/sparkx/Lattice3D.py [hunk 2]
```diff
@@ -1400,7 +1400,7 @@ def add_particle_data(
         particle_data: List[Particle],
         sigma: float,
         quantity: str,
-        kernel: str = "covariant",
```
Comment (NGoetz) [inline src/sparkx/Lattice3D.py:1403]: Again here I'd keep the default value, but would put "gaussian"

**Review by NGoetz — CHANGES_REQUESTED** [review]
Just minor changes.

File: src/sparkx/EventCharacteristics.py (hunk 1 above)
Comment (RenataKrupczak) [reply src/sparkx/EventCharacteristics.py]: That's done now

**Review by NGoetz — APPROVED** [review]
Thank you!

**Pull request merged**